didcomm/types/from_prior.py

```diff
@@ -0,0 +1,24 @@
+from typing import Optional, NamedTuple
+
+from didcomm.types.algorithms import SignAlg
+from didcomm.types.types import DID, KID, JWS
+
+
+class FromPrior(NamedTuple):
+    """from_prior property of a plaintext."""
+    iss: DID
+    sub: DID
+    aud: Optional[str] = None
+    exp: Optional[int] = None
+    nbf: Optional[int] = None
+    iat: Optional[int] = None
+    jti: Optional[str] = None
+
+    def as_jws(self, sign_alg: SignAlg, iss_kid: KID = None) -> JWS:
```

Comment:
I believe sign_alg is not needed as sign alg is taken from the DID DOC (by the curve type)